ocs_ci/helpers/keyrotation_helper.py

```diff
@@ -350,3 +351,48 @@ def compare_old_with_new_keys():
 
         log.info("Keyrotation is sucessfully done for the all OSD.")
         return True
+
+
+class PVKeyrotation(KeyRotation):
+    def __init__(self, sc_obj):
+        self.sc_obj = sc_obj
+        self.kms = get_kms_deployment()
+
+    def annotate_storageclass_key_rotation(self, schedule="@weekly"):
+        """
+        Annotate Storageclass To enable keyrotation for encrypted PV
+        """
+        annot_str = f"keyrotation.csiaddons.openshift.io/schedule='{schedule}'"
+        log.info(f"Adding annotation to the storage class:  {annot_str}")
+        self.sc_obj.annotate(annotation=annot_str)
+
+    def wait_till_keyrotation(self, device_handle, tries=5, delay=20):
+        """
+        Waits until the key rotation occurs for a given device handle.
+
+        Args:
+            device_handle (str): The handle or identifier for the device whose key
+                rotation is to be checked.
+
+        Returns:
+            bool: True if the key rotation is successful, otherwise False.
+        """
+        current_key = self.kms.get_pv_secret(device_handle)
+
+        @retry(UnexpectedBehaviour, tries=tries, delay=delay)
+        def compare_keys():
```

Comment:
this could be a separate function, which we can use in both `wait_till_keyrotation` and as a standalone one